### PR TITLE
Task/manager applicant review localizations

### PR DESCRIPTION
**Pull request by xander-miller**
Based on Issue #814

### CODE REVIEW

**Comment by xander-miller** [comment]
@gobyrne Ready for french localizations:

- [ ]  `resources/assets/js/components/ApplicationReview/localizations/fr.json`

**Comment by tristan-orourke** [comment]
Looks like if we want to make use of pluralization rules, we'll need to worry about [IE11 not supporting it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl). Should we be adding pollyfills proactively?

**Comment by Khristophor** [comment]
> Looks like if we want to make use of pluralization rules, we'll need to worry about [IE11 not supporting it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl). Should we be adding pollyfills proactively?

Do we have a target browser list anywhere? Whatever we need to support we should be polyfilling for, yes.

File: resources/assets/js/components/ApplicationReview/localizations/en.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "backToApplicantList": "< Save and Go Back to Applicant List",
+    "cancelButton": "Cancel",
+    "confirmButtion": "Confirm",
+    "screenOutConfirm": "Screen out the candidate?",
+    "screenInConfirm": "Put the candidate back in the \"under consideration\" section?",
+    "oops": "Oops...",
+    "somethingWrong": "Something went wrong while saving a review. Try again later.",
+    "copyEmailsButton": "Copy Emails",
+    "copied": "Copied!",
+    "daysSinceClosed": "EN {dayCount, plural, =0 {No Days} one {# Day} other {# Days}} Since Close"
```
Comment (Khristophor) [inline resources/assets/js/components/ApplicationReview/localizations/en.json]: Looks like the `EN` should be removed here?

File: resources/assets/js/components/ApplicationReview/ApplicationReview.tsx [hunk 2]
```diff
@@ -1,11 +1,110 @@
...
+  },
+  notReviewed: {
+    id: "notReviewed",
+    defaultMessage: "<default/> Not Reviewed",
+    description: "Decision dropdown label"
+  },
+  saving: {
+    id: "saving",
+    defaultMessage: "<default/> Saving...",
+    description: "Dynaming Save button label"
```
[the remark this answers is not in the document]
Comment (xander-miller) [reply resources/assets/js/components/ApplicationReview/ApplicationReview.tsx]: 👀 Yes it should. Fixing it now.

File: resources/assets/js/components/ApplicationReview/ApplicationReviewWithNav.tsx [hunk 3]
```diff
@@ -1,11 +1,110 @@
...
+    id: "decision",
+    defaultMessage: "<default/> Decision",
+    description: "Decision dropdown label"
+  },
+  notReviewed: {
+    id: "notReviewed",
+    defaultMessage: "<default/> Not Reviewed",
+    description: "Decision dropdown label"
+  },
+  saving: {
```
[the remark this answers is not in the document]
Comment (xander-miller) [reply resources/assets/js/components/ApplicationReview/ApplicationReviewWithNav.tsx]: most of this file is a repeat that needs to be refactored out.  The message declarations should just be on the one component once this is refactored.

File: resources/assets/js/components/ApplicationReview/ApplicationReviewWithNav.tsx [hunk 4]
```diff
@@ -1,11 +1,110 @@
...
+  },
+  emailCandidate: {
+    id: "emailCandidate",
+    defaultMessage: "<default/> Email this candidate.",
+    description: "Title, hover text, for email link."
+  },
+  viewApplicationTitle: {
+    id: "viewApplication.title",
+    defaultMessage: "<default/> View this applicant's application.",
+    description: "Title, hover text, for View Applection Link"
```
[the remark this answers is not in the document]
Comment (xander-miller) [reply resources/assets/js/components/ApplicationReview/ApplicationReviewWithNav.tsx]: 👀. Fixing it now.

File: resources/assets/js/components/ApplicationReview/ReviewApplications.tsx [hunk 5]
```diff
@@ -33,28 +34,53 @@ const ReviewApplications: React.StatelessComponent<ReviewApplicationsProps> = ({
...
-        "Review the applicants in the Veterans and Canadian Citizens section. If none or very few of these applicants meet the requirements, you can still consider non-Canadian Citizen applications in the Optional Considerations section",
+      title: {
+        id: "underConsideration.title",
+        defaultMessage: "<default/> Under Consideration",
+        description: "Under construction category title"
+      },
+      description: {
+        id: "underConsideration.description",
+        defaultMessage:
+          "<default/> Review the applicants in the Veterans and Canadian Citizens section. If none or very few of these applicants meet the requirements, you can still consider non- candian - citizen applications in the Optional Consideration section",
```
[the remark this answers is not in the document]
Comment (xander-miller) [reply resources/assets/js/components/ApplicationReview/ReviewApplications.tsx]: ✔️

File: resources/assets/js/components/ApplicationReview/ReviewApplications.tsx [hunk 6]
```diff
@@ -33,28 +34,53 @@ const ReviewApplications: React.StatelessComponent<ReviewApplicationsProps> = ({
 }: ReviewApplicationsProps): React.ReactElement => {
   const categories = [
     {
-      title: "Under Consideration",
-      description:
-        "Review the applicants in the Veterans and Canadian Citizens section. If none or very few of these applicants meet the requirements, you can still consider non-Canadian Citizen applications in the Optional Considerations section",
+      title: {
+        id: "underConsideration.title",
+        defaultMessage: "<default/> Under Consideration",
+        description: "Under construction category title"
```
[the remark this answers is not in the document]
Comment (xander-miller) [reply resources/assets/js/components/ApplicationReview/ReviewApplications.tsx]: You got me. I was subconsciously trying to make front end sh*t again. Fixed.

File: resources/assets/js/components/ApplicationReview/ReviewApplications.tsx [hunk 7]
```diff
@@ -33,28 +34,53 @@ const ReviewApplications: React.StatelessComponent<ReviewApplicationsProps> = ({
...
-      description: "These applications have already been screened out.",
+      title: {
+        id: "screenedOut.title",
+        defaultMessage: "<default/> No Longer Under Consideration",
+        description: "Screened out category title"
+      },
+      description: {
+        id: "screenedOut.description",
+        defaultMessage:
+          "<default/> These applications have allready been screened out.",
```
[the remark this answers is not in the document]
Comment (xander-miller) [reply resources/assets/js/components/ApplicationReview/ReviewApplications.tsx]: ✔️

File: resources/assets/js/components/ApplicationReview/ReviewCategory.tsx [hunk 8]
```diff
@@ -22,7 +28,24 @@ interface ReviewCategoryProps {
   prioritizeVeterans: boolean;
 }
 
-const ReviewCategory: React.StatelessComponent<ReviewCategoryProps> = ({
+const localizations = defineMessages({
+  confirmButton: {
+    id: "confirmButtion",
+    defaultMessage: "<default/> Confirm",
+    description: "Confirm buttion for modal dialogue boxes"
```
[the remark this answers is not in the document]
Comment (xander-miller) [reply resources/assets/js/components/ApplicationReview/ReviewCategory.tsx]: ✔️

File: resources/assets/js/components/ApplicationReview/ReviewCategory.tsx [hunk 9]
```diff
@@ -22,7 +28,24 @@ interface ReviewCategoryProps {
...
+  },
+  screenOutAllConfirm: {
+    id: "screenOutAll.confirm",
+    defaultMessage:
+      "<default/> Are you sure you want to screen out all Optional candidates?",
+    description:
+      "Confirm dialogue test for screening out all optional candidates."
```
[the remark this answers is not in the document]
Comment (xander-miller) [reply resources/assets/js/components/ApplicationReview/ReviewCategory.tsx]: ✔️

File: resources/assets/js/components/ApplicationReview/localizations/en.json (hunk 1 above)
Comment (xander-miller) [reply resources/assets/js/components/ApplicationReview/localizations/en.json]: ✔️

**Pull request merged**